src/app/features/room/RoomViewHeader.tsx

```diff
@@ -415,6 +460,7 @@ export function RoomViewHeader() {
               )}
             </TooltipProvider>
           )}
+
```

Comment:
Move "Chat" button in front of "Options" to have a consistent button order with text rooms. Also fill icon when chat is open to prepare for #2367 
```suggestion

          {room.isCallRoom() && !isDirectMessage() && (
            <TooltipProvider
              position="Bottom"
              offset={4}
              tooltip={
                <Tooltip>
                  <Text>Chat</Text>
                </Tooltip>
              }
            >
              {(triggerRef) => (
                <IconButton ref={triggerRef} onClick={toggleChat}>
                  <Icon size="400" src={Icons.Message} filled={isChatOpen} />
                </IconButton>
              )}
            </TooltipProvider>
          )}

```